feature/experimental/bgp/ate_tests/base_bgp_session_parameters/README.md

```diff
@@ -37,7 +41,6 @@ Test the normal session establishment and termination:
     And include the following session parameters for all cases:
 
     *   Explicitly specified Router ID.
-    *   Enable MD5 authentication on DUT and ATE.
     *   Explicit holdtime interval and keepalive interval.
     *   Explicit connect retry interval.
 
```

Comment:
```suggestion

Test the password authentication on session establishment:

*   Configure matching passwords on DUT and ATE. Verify that BGP adjacency succeeds.
*   Configure mismatched passwords on DUT and ATE. Verify that BGP adjacency fails.
```

Reply:
This should be done in its own function in test code:

```
func TestPassword(t *testing.T) {
  ...
}
```

Reply:
updated , thank you